main.tf

```diff
@@ -19,20 +19,22 @@
  *****************************************/
 locals {}
 
-resource "google_bigquery_dataset" "default" {
-  dataset_id                  = "${var.dataset_id}"
-  friendly_name               = "${var.dataset_name}"
-  description                 = "${var.description}"
+resource "google_bigquery_dataset" "main" {
+  dataset_id    = "${var.dataset_id}"
+  friendly_name = "${var.dataset_name}"
+  description   = "${var.description}"
+
   #TODO: add if condition to validate if neither US or EU are supplied
-  location                    = "${var.region}"
+  location = "${var.region}"
+
   #TODO: format this ne excluded by default but can optionally be defined if the user wishes
   default_table_expiration_ms = "${var.expiration}"
```

Comment:
This should be an integer value, not a string (https://www.terraform.io/docs/providers/google/r/bigquery_dataset.html)